utils/chaos_ai/docker/config/yml/qtable.yml

```diff
@@ -0,0 +1,15 @@
+
+    Get QTable from the Chaos ID.---

```

Comment:
Using the apidocs I was able to exeute the "robot-shop" example on my kubeconfig of an openshift cluster. When I tested out the q table I got an error that said the chaos id was invalid with a 200 return code. I'm not super clear what this api should return but unsure why the id was invalid. NOTE: I did try it out in the other get calls with no issues